scripts/build.py

```
@@ -0,0 +1,106 @@
+import os
+import subprocess
+import hashlib
+import tarfile
```

Ruff (0.11.9)

`tarfile` imported but unused

Remove unused import: `tarfile`

(F401)